.vscode/settings.json

```diff
@@ -5,4 +5,8 @@
             "mode": "auto"
         }
     ],
+    "jest.runMode": {
+        "type": "on-save",
```

Comment:
Found this more reliable than the default 'watch' for running the tests whilst making changes.